protocol/v2/ssv/runner/runner.go

```diff
@@ -82,7 +84,7 @@ func (b *BaseRunner) MarshalJSON() ([]byte, error) {
 		State              *State
 		Share              map[phase0.ValidatorIndex]*spectypes.Share
 		QBFTController     *controller.Controller
-		BeaconNetwork      spectypes.BeaconNetwork
+		NetworkConfig      networkconfig.NetworkConfig
```

Comment:
is it really needed or we could do with just beacon network and domaintype?

Reply:
I think it needs only domain type and `EstimatedEpochAtSlot` but I think it's easier to just pass network config, maybe we'll need something else in the future